src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetVerbCommand.cs

```diff
@@ -71,7 +71,20 @@ protected override void ProcessRecord()
 
                         VerbInfo verb = new VerbInfo();
                         verb.Verb = field.Name;
+
+                        FieldInfo aliasField = typeof(VerbAliasPrefixes).GetField(field.Name);
+                        if(aliasField != null)
+                        {
+                            verb.AliasPrefix = (string)aliasField.GetValue(null);
+                        }
+
                         verb.Group = groupName;
+
+                        FieldInfo descriptionField = typeof(VerbDescriptions).GetField(field.Name);
```

Comment:
The description lookup should be a static method in the VerbDescription class; especially since the strings should be in a resx file to enable localization.